dnf-docker-test/features/reinstall-1.feature

```diff
@@ -1,4 +1,4 @@
-Feature: reinstall
+Feature: Reinstall won't break dependencies
```

Comment:
that's not true ;)

Something like `Re-install must not change "reason" of package which has been re-installed`